src/styles/about.scss

```diff
@@ -1,47 +1,89 @@
-img {
-    margin-top: 0.5rem;
-    width: auto;
-    width: 40vh;
-    margin-right: 3rem;
-}
-
 .about-the-author {
+  margin-bottom: 1rem;
+
+  @media (min-width: 700px) {
     display: flex;
     justify-content: space-between;
-    padding: 2rem 30rem 10rem 30rem;
```

Comment:
These are big numbers. 30 + 30 * 16px (aka 1rem) = 960px of horizontal space. That much padding is going to give you trouble in most situations. It would be better to find a flexbox solution for a design like this.